alot/db/message.py

```diff
@@ -107,13 +107,11 @@ def get_thread_id(self):
         return self._thread_id
 
     def get_message_parts(self):
-        """returns a list of all body parts of this message"""
+        """yield all body parts of this message"""
         # TODO really needed? email  iterators can do this
```

Comment:
yes you are right. A quick "fix" would be to just remove the comment.
But this function is actually only used once, and there we could instead also
use `https://docs.python.org/2/library/email.iterators.html#email.iterators.typed_subpart_iterator`..